contents/docs/libraries/ios/_snippets/autocapture.mdx

```diff
@@ -0,0 +1,74 @@
+
+> 🚧 **Note:** `$autocapture` is currently supported only in UIKit and remains in an experimental state, with the feature disabled by default. SwiftUI support is planned for an upcoming release.
+
+### Autocapture configuration
+
+You can enable or disable Autocapture through the `PostHogConfig` object.
+
+```swift file=AppDelegate.swift
+let POSTHOG_API_KEY = "<ph_project_api_key>"
+// usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+let POSTHOG_HOST = "https://us.i.posthog.com"
+
+let config = PostHogConfig(apiKey: POSTHOG_API_KEY, host: POSTHOG_HOST)
+
+// capture screen views
+config.captureScreenViews = true 
+// capture application lifecycle events (installed, updated, opened, backgrounded)
+config.captureApplicationLifecycleEvents = true
+// capture element interactions (button presses, text input changes, etc.)
+config.captureElementInteractions = true // TIP: Disabled by default
+
+PostHogSDK.shared.setup(config)
+```
+
+### Capturing screen views
+
+With [`configuration.captureScreenViews`](/docs/libraries/ios#all-configuration-options) set as `true`, PostHog will try to record all screen changes automatically.
+
+If you want to manually send a new screen capture event, use the `screen` function.
+
+```swift
+PostHogSDK.shared.screen("Dashboard", properties: ["fromIcon": "bottom"])
+```
+
+> **Important:** Please note that `captureScreenViews` configuration is intented for `UIKit` use. For `SwiftUI` screen captures, please see the following section.
+
+### Capturing screen views in SwiftUI
+
+To track a screen view in `SwiftUI`, apply the `postHogScreenView` modifier to your full-screen views. 
+PostHog will send a `$screen` event when `onAppear` action is executed and will infer a screen name based on the view’s type. You can provide a custom name and event properties if needed.
+
+```swift file=HomeView.swift
+// This will trigger a screen view event with $screen_name: "HomeViewContent"
+struct HomeView: View {
+    var body: some View {
+        HomeViewContent()
+            .postHogScreenView()
+    }
+}
+
+// This will trigger a screen view event with $screen_name: "My Home View" and an additional event property from_button: "start"
+struct HomeView: View {
+    var body: some View {
+        HomeViewContent()
+            .postHogScreenView("My Home View", ["from_button": "start"])
+    }
+}
+```
+
+*In SwiftUI, views can range from entire screens to small UI components. Unlike UIKit, SwiftUI doesn’t clearly distinguish between these levels, which makes automatic tracking of full-screen views harder.*
+
+### Disabling autocapture
+
+To disable Autocapture, set `config.autocapture` to `false` in the configuration.
+
+> **Note:** Disabling `$autocapture` will *not* prevent the collection of screen view and application lifecycle events. To disable these specific events, use `config.captureScreenViews` and `config.captureApplicationLifecycleEvents` in the configuration.
```

Comment:
this note has to be changed if `config.autocapture` does not exist anymore as well